gnomad/resources/grch37/reference_data.py

```diff
@@ -10,6 +10,79 @@
     import_sites_vcf,
 )
 
+
+def _import_gencode_cds(gtf_path: str, **kwargs) -> hl.Table:
+    """
+    Get CDS intervals from GENCODE GTF file.
+
+    :param gtf_path: Path to GENCODE GTF file.
+    :return: Table with CDS intervals.
+    """
+    ht = hl.experimental.import_gtf(
+        gtf_path,
+        **kwargs,
+    )
+    ht = ht.annotate(
+        gene_id=ht.gene_id.split("\\.")[0],
+        transcript_id=ht.transcript_id.split("\\.")[0],
+    )
+    ht = (
+        ht.filter((ht.feature == "CDS") & (ht.transcript_type == "protein_coding"))
+        .select("gene_id", "transcript_id")
+        .distinct()
+    )
```

Comment:
Move this to another function that extracts the CDS info, this is similar to how we handled the clinvar import and filtering to pathogenic variants. We want to keep everything in the resource, but we can filter to or extract the specific info we want with another function

Reply:
For now, I also made it a resource, should we just put that function inside transcript_annotation.py?